libs/web3/src/lib/use-ethereum-config.ts

```diff
@@ -23,27 +23,17 @@ export interface EthereumConfig {
   };
 }
 
-export const NETWORK_PARAMS_QUERY = gql`
-  query NetworkParamsQuery {
-    networkParameters {
-      key
-      value
-    }
-  }
-`;
-
 export const useEthereumConfig = () => {
-  const { data, loading, error } =
-    useQuery<NetworkParamsQuery>(NETWORK_PARAMS_QUERY);
+  const { data, loading, error } = useNetworkParamsQuery();
```

Comment:
suggestion (non blocking): Avoid fetching all params by using the singular `useNetowrkParam` hook to just retrieve the Ethereum config?